src/input/datetime.rs

```diff
@@ -131,6 +131,93 @@ impl EitherTimedelta<'_> {
             Self::PySubclass(py_timedelta) => pytimedelta_subclass_as_duration(py_timedelta),
         }
     }
+
+    pub fn total_seconds(&self) -> PyResult<f64> {
```

Comment:
This has been copied from the original time we tried to implement float seconds/milliseconds into the lib 😄